build.sbt

```diff
@@ -95,6 +95,8 @@ val commonSettings = Sonatype.sonatypeSettings ++ assemblySettings ++ Seq(
 
   // protobuf-lite is an older subset of protobuf-java and causes issues
   excludeDependencies += "com.google.protobuf" % "protobuf-lite",
+  // hamcrest-core overlaps with hamcrest-all and causes issues in IntelliJ
+  excludeDependencies += "org.hamcrest" % "hamcrest-core",
```

Comment:
Is this included due to stale branch? Rebase on master?

Reply:
I rebased on master

Reply:
bad rebase apparently.